package.json

```
@@ -39,7 +39,7 @@
   },
   "devDependencies": {
     "@argos-ci/core": "1.5.5",
-    "@mui/monorepo": "github:mui/material-ui#f3cb496c999acbc8f19533e38df20be12e56d059",
+    "@mui/monorepo": "github:mui/material-ui#bbfb2cd742ed3aca3d0c8f6dc3e890ed4fc1d7f2",
```

Should be 

```suggestion
    "@mui/monorepo": "github:mui/material-ui#next",
```

here, otherwise, renovate will try to use the default branch, master

---

Actually, my point is not true, it looks like Renovate uses the GitHub repository default branch. Let's do the opposite, it looks clearer and avoid the need to update the branch as it changes: https://github.com/mui/mui-x/commit/906ad7d286e7fa0663047cc90543d287e6da41ad.

---

Yeah, this has actually worked super well for us so far.